rgw/v2/tests/s3_swift/configs/test_sts_using_boto.yaml

```diff
@@ -1,10 +1,33 @@
 # polarion test case id: CEPH-83572938
 config:
- bucket_count: 2
- objects_count: 5
- objects_size_range:
-  min: 5
-  max: 15
- test_ops:
-      create_bucket: true
-      create_object: true
+     bucket_count: 2
+     objects_count: 5
+     objects_size_range:
+          min: 5
+          max: 15
+     test_ops:
+          create_bucket: true
+          create_object: true
+     sts:
+          policy_document:
+               "Version": "2012-10-17"
+               "Statement":
+                    [
+                         {
+                              "Effect": "Allow",
+                              "Principal":
+                                   {
+                                        "AWS":
+                                             ["arn:aws:iam:::user/<user_name>"],
+                                   },
+                              "Action": ["sts:AssumeRole"],
+                         },
+                    ]
+          role_policy:
+               "Version": "2012-10-17"
+               "Statement":
+                    {
+                         "Effect": "Allow",
+                         "Action": "s3:*",
+                         "Resource": "arn:aws:s3:::*",
+                    }
```

Comment:
During the loading of the values, we could opt to use YAML instead of JSON. In this way, we could conformant to YAML

Reply:
yaml is loaded initially [here](https://github.com/red-hat-storage/ceph-qe-scripts/blob/master/rgw/v2/lib/resource_op.py#L177l) using `safeload`.  the reason i used json here is that, sts policies are not dict, they are string which are in this format specifically. `'{"policy": "<some_policy"}'` including the placements of double quotes and single quotes. the easiest was to use json. STS wont consider any other format.

Reply:
okay, I see the reason why we need to do it this way.